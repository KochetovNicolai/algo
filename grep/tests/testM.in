[a-z]+[0-9]+[A-Z]+
abc42DEF
!a01039019380941518452109Z-
a0a0a
a0a0B
Abc00FF