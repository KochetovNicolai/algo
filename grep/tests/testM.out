3	6 7 8 
3	6 7 8 
3	6 7 8 
0	
0	
0	
0	
0	

0	
1	26 
0	
0	
0	
0	
0	
0	
0	
0	
0	
0	
0	
0	
0	
0	
0	
0	
0	
0	
0	
0	
0	
0	
0	
0	
0	

0	
0	
0	
0	
0	

0	
0	
1	5 
0	
0	

0	
2	6 7 
2	6 7 
0	
0	
0	
0	

